lib/statsd.js

```diff
@@ -42,26 +43,55 @@ var Client = function (host, port, prefix, suffix, globalize, cacheDns, mock,
       maxBufferSize : maxBufferSize,
       bufferFlushInterval: bufferFlushInterval,
       telegraf    : telegraf,
-      sampleRate    : sampleRate
+      sampleRate  : sampleRate,
+      protocol    : protocol
     };
   }
```

Comment:
Someone is going to do "TCP" at some point and get confused.  How about adding:
```
if (options.protocol) {
  options.protocol = options.toLowerCase();
}
```

Reply:
Addressed in https://github.com/brightcove/hot-shots/pull/79/commits/fe2d446bf926e8b5107804690e275c431b2ca3fc.